packages/app/src/cli/prompts/config.ts

```diff
@@ -83,8 +83,8 @@ export async function confirmPushChanges(options: PushOptions, app: App) {
     configuration.access_scopes.scopes = getAppScopesArray(configuration).join(',')
 
   const [updated, baseline] = deepDifference(
-    {...(rewriteConfiguration(AppSchema, configuration) as object), build: undefined},
-    {...(rewriteConfiguration(AppSchema, remoteConfiguration) as object), build: undefined},
+    {...(rewriteConfiguration(AppSchema, configuration) as object), build: undefined, access: undefined},
+    {...(rewriteConfiguration(AppSchema, remoteConfiguration) as object), build: undefined, access: undefined},
```

Comment:
Setting `access: undefined` is our way "ignoring" the `[access]` section when diffing the local and remote configs.

Reply:
This works for now, but as we add more modules to the app.toml this won't scale. Let's keep it for now but we'd need to think of a way to improve it.
For example, each specification could define which fields it uses from the app toml, and we can ignore here all fields that are used in specifications.